app/kubemci/pkg/gcp/backendservice/backendservicesyncer_test.go

```diff
@@ -109,7 +154,7 @@ func TestDeleteBackendService(t *testing.T) {
 			Port: port,
 			Name: portName,
 		},
-	}, []string{igLink}); err != nil {
+	}, []string{igLink}, false /*forceUpdate*/); err != nil {
```

Comment:
Should the `/*forceUpdate*/` be removed?

Reply:
Probably not. It is poor man's keyword argument.

Although, I must admit that I find this form of complex object construction directly in a function call statement absurd.

Reply:
+1 to keeping it as documentation